src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1073,6 +1079,11 @@ void program_node::init_onednn_primitive_attributes() {
                     attrs->set_output_scales(mask, {DNNL_RUNTIME_F32_VAL});
                     update_onednn_post_op_list(onednn_post_op_type::scale, dep_idx);
                 }
+            } else {
+                std::stringstream error_msg;
+                error_msg << "Unsupported eltwise mode: " << static_cast<int>(desc.typed_desc<eltwise>()->mode) << ". ";
+                error_msg << desc.desc->id << " is fused node of " + this->id() + ".";
+                OPENVINO_ASSERT(false, error_msg.str());
```

Comment:
could you check that there is no diff from dryrun or partial run with this change?

Reply:
No difference at dryrun test.